### PR TITLE
Tool builder: add profile and suffix version

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

File: planemo/tool_builder.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 DEFAULT_CWL_VERSION = "v1.0"
 
 
-TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}" python_template_version="3.5">
+TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}+galaxy0" python_template_version="3.5" profile="20.01">
```
Comment (nsoranzo) [inline planemo/tool_builder.py:22]:
```suggestion
TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}+galaxy0" python_template_version="3.5" profile="21.05">
```

I'd use the oldest release currently supported.

Comment (gallardoalba) [reply planemo/tool_builder.py]: Is it necessary to modify the build/lib/planemo/tool_builder.py file?

**Review by nsoranzo — APPROVED** [review]

File: planemo/tool_builder.py (hunk 1 above)
Comment (nsoranzo) [reply planemo/tool_builder.py]: Nope, that's not part of the repository.

File: planemo/tool_builder.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 DEFAULT_CWL_VERSION = "v1.0"
 
 
-TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}" python_template_version="3.5">
+TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}+galaxy0" python_template_version="3.5" profile="21.05">
```
Comment (mvdbeek) [inline planemo/tool_builder.py:22]: We can also drop the `python_template_version`, that is implied by a recent profile

**Pull request merged**